governing-documents/project-annual-review.md

```diff
@@ -0,0 +1,92 @@
+
+### Filing an Annual Review
+Hyperledger Foundation staff will notify the project maintainers and copy the TOC when the project review is due. 
+
+Project maintainers are responsible for agreeing between them who will complete the annual review. One of the maintainers should create the review in GitHub under [hyperledger/toc/project-reports/](https://github.com/hyperledger/toc/tree/gh-pages/project-reports).
+
+* Raise a PR titled *[Project name] [year] Annual Review* (e.g., Amazing Project 2024 Annual Review)
+* The PR should include a file called `./<year>/<year>-<project name>-annual.md` (e.g., `2024/2024-amazingproj-annual.md`) with the [contents described below](#annual-review-contents)
+* Send an email to the [TOC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it.
+
+**If your annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TOC is likely to decide to archive the project and move it to End of Life status.**
+
+**NOTE:** If a project has genuinely stalled we can save everyone’s time and effort by archiving it.
+
+### Annual Review Contents
+Your annual review should answer the following questions: 
+
+* Include a link to your project’s [LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/projects). We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add color to the numbers and graphs we will see on Insights.
+* How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing [MAINTAINERS file](../guidelines/MAINTAINERS-guidelines.md) if appropriate.)
+* What do you know about adoption, and how has this changed since your last review or since being accepted into Hyperledger Foundation? If you can list companies that are adopters of your project, please do so. Feel free to link to an existing ADOPTERS file if appropriate.
+* How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)
+* What are the goals for the next year of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation? 
+* How can the Hyperledger Foundation help you achieve your upcoming goals? 
+* Do you think that your project meets the criteria for another stage (see [Project Lifecycle](./project-lifecycle.md))?
+
+## Potential Outcomes
+The outcome of the annual review is either:
+
+* At least two-thirds of the TOC members agree to continue the project at its current stage, or
+* If enough TOC members do not agree to continue the project at its current stage, we will discuss with you what stage might be the appropriate next stage, including End of Life stage.
```

Comment:
I think the decision-making follows a different path if the maintainers have explicitly asked for a stage transition (which typically will be Incubation --> Graduated). In that case, the TOC members should be asked to vote on whether or not they approve the transition. If the maintainers have not asked for a transition, then the TOC members can decide according to the order specified in Lines 60 and 61.

Reply:
Agree. This is what the `NOTE` after this is trying to say. That we will use existing processes to handle lifecycle changes.